documentation/src/main/asciidoc/user_guide/eviction.adoc

```diff
@@ -170,18 +172,45 @@ stay in the cache or _maximum idle time_ , time it's allowed to be untouched
 
 .  _Expiration_ is a top-level construct, represented in the configuration as well as in the cache API.
 .  While eviction is _local to each cache instance_ , expiration is _cluster-wide_ .  Expiration lifespans and maxIdle values are replicated along with the cache entry.
-.  While maxIdle is replicated, expiration due to maxIdle is not cluster wide, only lifespan.  As such it is not recommended to use maxIdle in a clustered cache.
+.  Max idle requires additional network messages to work in a cluster.  As such using maxIdle in a clustered cache can cause some operations to be slower at times.
```

Comment:
I'd say it *will* cause some operations to be slower at times.

Reply:
Okay :)